frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/types.ts

```diff
@@ -110,3 +110,8 @@ export type VMSettingsValidationConfig = {
     validator: VmSettingsValidator;
   };
 };
+
+export type OperatingSystemArray = {
```

Comment:
please do not create array types, also it should not be under wizard directory

Reply:
done.